corehq/apps/export/models/deid_export.py

```diff
@@ -18,11 +18,11 @@ class Meta:
         ]
 
     @classmethod
-    def get_deid(cls, value, doc, domain=None):
-        if doc is not None:
-            # use domain from the couch doc if one was passed in
+    def get_deid(cls, value, doc):
+        if isinstance(doc, dict):
             domain = doc['domain']
-
+        else:
+            domain = doc.domain
```

Comment:
Probably good to have a test cover this code as well.

Reply:
https://github.com/dimagi/commcare-hq/pull/35772/commits/918c24a426bf239b8cdc5a58d1ef900a2a5e47f5